build/schematics.ts

```diff
@@ -1,6 +1,20 @@
 import { exec } from './util';
+import * as fs from 'fs';
+import path = require('path');
```

Comment:
use `import * as path from 'path';` instead of `require`

Reply:
weird that it automatically used `require`..- fixing :)